tests/h/services/list_organizations_test.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import pytest
+
+from h.services.list_organizations import ListOrganizationsService
+from h.services.list_organizations import list_organizations_factory
+from h.services.organization import organization_factory
+
+
+class TestListOrganizations(object):
+
+    def test_returns_organizations_if_no_user(self, svc, organizations, alternate_organizations):
```

Comment:
It's likely you cribbed these tests from the `list_groups` service? In that service, the presence of a `user` affects the returned list of groups. However, the `list_organizations` doesn't know anything about users (I don't think)? So I'm not sure what this test on the one following it test.

Reply:
Oh good call! I forgot to adjust the names on those...it's really testing for the difference in authorities. (using the default authority of the service vs specifying your own.)